src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Controllers/AdminController.cs

```diff
@@ -432,14 +432,15 @@ public async Task<ActionResult> ForceDelete(ElasticIndexSettingsViewModel model)
         return RedirectToAction(nameof(Index));
     }
 
-    public async Task<IActionResult> Mappings(string indexName)
+    public async Task<IActionResult> IndexInfo(string indexName)
```

Comment:
So this was named Mappings because that's how ElasticSearch calls it in its terminology.

https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html

Reply:
So, need to see what is the difference between Mappings and doing a GetIndexInfo

Reply:
Can you send me a screenshot of the resulting textfield in the Admin UI so that I can compare?

Reply:
![image](https://github.com/user-attachments/assets/77e3fccd-9253-4a1c-adbc-b3a345efa7c9)
![image](https://github.com/user-attachments/assets/4cb78a43-c004-4bd7-b5a9-5751dd7026ae)

Sure, here it is. The whole idea is to see that analyzers and filters were applied.

Reply:
{
  "default_newind": {
    "aliases": {},
    "mappings": {
      "_meta": {
        "last_task_id": 0
      },
      "_source": {
        "excludes": [
          "Content.ContentItem.DisplayText.Analyzed"
        ]
      },
      "dynamic_templates": [
        {
          "*.Inherited": {
            "path_match": "*.Inherited",
            "match_mapping_type": "string",
            "mapping": {
              "type": "keyword"
            }
          }
        },
        {
          "*.Ids": {
            "path_match": "*.Ids",
            "match_mapping_type": "string",
            "mapping": {
              "type": "keyword"
            }
          }
        }
      ],
      "properties": {
        "Content": {
          "properties": {
            "ContentItem": {
              "properties": {
                "ContainedPart": {
                  "properties": {
                    "Ids": {
                      "type": "keyword"
                    },
                    "Order": {
                      "type": "float"
                    }
                  }
                },
                "ContentType": {
                  "type": "keyword"
                },
                "DisplayText": {
                  "properties": {
                    "Analyzed": {
                      "type": "text"
                    },
                    "Normalized": {
                      "type": "keyword"
                    },
                    "keyword": {
                      "type": "keyword"
                    }
                  }
                },
                "FullText": {
                  "type": "text"
                },
                "Owner": {
                  "type": "keyword"
                }
              }
            }
          }
        },
        "ContentItemId": {
          "type": "keyword"
        },
        "ContentItemVersionId": {
          "type": "keyword"
        }
      }
    },
    "settings": {
      "index": {
        "routing": {
          "allocation": {
            "include": {
              "_tier_preference": "data_content"
            }
          }
        },
        "number_of_shards": "1",
        "provided_name": "default_newind",
        "creation_date": "1728302642111",
        "analysis": {
          "filter": {
            "russian_stemmer": {
              "type": "stemmer",
              "language": "russian"
            },
            "english_stemmer": {
              "type": "stemmer",
              "language": "russian"
            },
            "russian_stop": {
              "type": "stop",
              "stopwords": "_russian_"
            },
            "russian_synonyms": {
              "type": "synonym_graph",
              "synonyms": [
                "\u0437\u043E\u043B\u043E\u0442\u043E, AU, XAU",
                "\u043F\u043E\u0437\u0438\u0446\u0438\u0438, \u0438\u043D\u0442\u0435\u0440\u0435\u0441"
              ]
            },
            "english_stop": {
              "type": "stop",
              "stopwords": "_russian_"
            }
          },
          "analyzer": {
            "default": {
              "filter": [
                "lowercase",
                "russian_stop",
                "russian_synonyms",
                "russian_stemmer",
                "english_stop",
                "english_stemmer"
              ],
              "type": "custom",
              "tokenizer": "standard"
            }
          }
        },
        "number_of_replicas": "1",
        "uuid": "V-oB0lgQRmWMiKQCJLLLTg",
        "version": {
          "created": "7160299"
        }
      }
    }
  }
}

Reply:
> So, need to see what is the difference between GetIndexMappings and GetIndexInfo.

The difference now, is that shows not only "mappings" section but full index information

Reply:
Ok for IndexInfo then